test/server/cards/02_SHD/events/DesperateAttack.spec.ts

```diff
@@ -0,0 +1,41 @@
+describe('Desperate Attack', function() {
+    integration(function(contextRef) {
+        describe('Desperate Attack\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['desperate-attack'],
+                        groundArena: [{ card: 'isb-agent', damage: 1 }, { card: 'cell-block-guard', damage: 1 }, { card: 'wampa', exhausted: true }],
```

Comment:
Any reason for wampa to be exhausted?

Reply:
Good question, I'll say it made sense at the time 😆

Reply:
No worries at all! We just try to keep the test setup as relevant to the test cases as possible so I always check